Symfony/CS/Tests/Fixer/Symfony/AliasFunctionsFixerTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Symfony;
+
+use Symfony\CS\Fixer\Symfony\AliasFunctionsFixer;
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+class AliasFunctionsFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideCases()
+    {
+        $cases = array();
+        foreach (AliasFunctionsFixer::$aliases as $alias => $master) {
+            /* valid cases */
+            $cases[] = array('<?php $smth->'.$alias.'($a);');
+            $cases[] = array('<?php '.$alias.'Smth($a);');
+            $cases[] = array('<?php smth_'.$alias.'($a);');
+            $cases[] = array('<?php new '.$alias.'($a);');
+            $cases[] = array('<?php new Smth\\'.$alias.'($a);');
+            $cases[] = array('<?php Smth::'.$alias.'($a);');
+            $cases[] = array('<?php new '.$alias.'\smth($a);');
+            $cases[] = array('<?php '.$alias.'::smth($a);');
+            $cases[] = array('<?php '.$alias.'\smth($a);');
+            $cases[] = array('<?php \\'.$alias.'($a);');
```

Comment:
This should be fixed. It is the only case where we really know that the global function is called. ;)